activerecord/test/cases/schema_dumper_test.rb

```diff
@@ -61,7 +61,7 @@ def test_schema_dump
 
   def test_schema_dump_uses_force_cascade_on_create_table
     output = dump_table_schema "authors"
-    assert_match %r{create_table "authors", force: :cascade}, output
+    assert_match %r{create_table "authors", .*?force: :cascade}, output
```

Comment:
Omitting will not have the same problem of changing the default without the compatibility layer? If a Rails 4 user load the schema in Rails 5 all ids will become bigserial.